app/persistence/entity/lock-event.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package entity
+
+import "database/sql"
+
+// TODO: Put proper GORM entity model here
+
+type LockEvent struct {
+	Id            string `gorm:"primaryKey"` // represents {ethTxHash}-{logIndex}
+	ScheduleID    string
```

Comment:
We will have 2 Scheduled Transactions created. The first one is `Mint` and the second one `Transfer`. I think that we will need to have `ScheduleMintID` and `ScheduleTransferID`